src/main/java/ConnectionManagement/Connection.java

```diff
@@ -0,0 +1,14 @@
+package ConnectionManagement;
```

Comment:
Try to built the habit of prefixing your package names with lastname.firstname or your reversed domain if you own one (for instance, I often use either `io.zipcoder` or `com.davidginzberg` as my prefix, depending on whether it is a ZCW or personal project). This makes it less likely that you'll accidentally collide with someone else's codebase in collaborative projects.